### PR TITLE
PD-206 -pull CSI Quay Images

### PR DESCRIPTION
**Pull request by andreipope**

### CODE REVIEW

File: content/portworx-install-with-kubernetes/on-premise/airgapped/_index.md [hunk 1]
```diff
@@ -1,66 +1,80 @@
...
     ```text
-    docker save $PX_IMGS $PX_ENT | ssh intranet-host docker load
+    docker save $PX_IMGS $PX_ENT | ssh <intranet-host> docker load
     ```
 
-## Step 2: Loading Portworx images on your nodes
+    For `<intranet-host>`, use the address of your node.
 
-If you have nodes which have access to a private registry, follow [Step 2a: Push to local registry server, accessible by air-gapped nodes](#step-2a-push-to-local-registry-server-accessible-by-air-gapped-nodes).
+    Note that the above command uses `ssh` to load the images on a node called `intranet-host`. If your cluster nodes don't have Internet access, you first need to copy over the images to one of the nodes using a tarball.
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/on-premise/airgapped/_index.md]:
```suggestion
{{<info>}}
    Note that the above command uses `ssh` to load the images on a node called `intranet-host`. If your cluster nodes don't have Internet access, you first need to copy over the images to one of the nodes using a tarball.
{{</info>}}
```

**Comment by andreipope** [comment]
@nathan-px Thanks for taking the time to review this PR.

**Comment by andreipope** [comment]
@luayalem @lpabon @harsh-px Hey guys, please review this PR when you get the chance.

**Comment by andreipope** [comment]
@luayalem Can you please review this PR?

**Comment by andreipope** [comment]
@luayalem Just wanted to bump this to your attention.

**Pull request merged**